lib/conduit_lwt_tls.mli

```diff
@@ -32,9 +32,10 @@ module Server : sig
     Tls.Config.server ->
     Lwt_unix.file_descr ->
     (Lwt_unix.file_descr * Lwt_io.input_channel * Lwt_io.output_channel) Lwt.t
+  [@@@deprecated "Low level function. Will be removed."]
```

Comment:
Is this function is the only way to pass in a `Tls.Config.server`? That seems quite important.